packages/wonder-blocks-dropdown/components/action-menu.js

```diff
@@ -176,6 +179,7 @@ export default class ActionMenu extends React.Component<MenuProps, State> {
             <ActionMenuOpener
                 disabled={disabled}
                 onClick={() => this.toggleMenu()}
+                ref={node => {this.openerNode = ReactDOM.findDOMNode(node)}}
```

Comment:
We are using the ref on the opener to determine its position later with getBoundingClientRect.